### DISCUSSION TITLE
BasePath didn't affeat components' Links

### DISCUSSION THREAD

**Discussion by LintaoAmons** [themesberg/flowbite-react#240] [Q&A] [answered]
By using <Link> which provided by nextjs, `next.config.js` could set `BasePath` to add a prefix to the `href`.

But components provided by `flowbite` seems didn't take `BasePath` into account.

For example: `Navbar.Link` won't add `basePath` as a prefix, thus need to hard code the prefix to the `href` attribute.

**Comment by rluders** [comment]
@tulup-conner you know more about nextjs, remix and that stuff that I do. So... probably the best person to think about it.

**Reply by tulup-conner** [reply]
I have no idea TBH. I'm not sure how the library could respect something from a Next config file.

**Comment by tulup-conner** [comment, marked as answer]
To resolve this in your usage you should probably try to use `next/link` where possible instead of the `href` built into `flowbite-react`, which is just a normal `<a>`. There's a discussion about that here: https://github.com/themesberg/flowbite-react/discussions/592
